package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-native-windows-repo",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "change": "node ./node_modules/beachball/bin/beachball.js change",
+    "checkchange": "node ./node_modules/beachball/bin/beachball.js check",
+    "clean": "lerna run clean"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:microsoft/react-native-windows.git"
+  },
+  "license": "MIT",
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "vnext"
```

Comment:
should current be in here?

Reply:
No.  Current will not be manged by beachball, or yarn.  Since current and vnext both have the same package names, it confuses all the monorepo tools.